semcore/d3-chart/src/a11y/insights.ts

```diff
@@ -302,7 +302,7 @@ export const extractDataInsights = (
                   to: shortMovingAverage[i],
                 },
                 width: i - lastSwitch,
-                label: { from, to, dataKey: valueKey },
+                label: { from, to, dataKey: hints.titles.valuesAxes[valueKey] ?? valueKey },
```

Comment:
I think it would be more correct to extend the label object 
instead of `{from: ..., to: ..., dataKey: ...}` make `{from: ..., to: ..., dataKey: ..., value: string}`, put the `hints.titles.valuesAxes[valueKey]` into `value` and also read `value ?? dataKey` in serializer cause `dataKey` should be something like `x`, `y`, `y0`, `y1` etc. that might be read in data array

Reply:
changed, but named displayName